transport/http/config_test.go

```diff
@@ -499,3 +501,43 @@ func useFakeBuildClient(t *testing.T, want *wantHTTPClient) TransportOption {
 		return buildHTTPClient(options)
 	})
 }
+
+func TestFallbackHandler(t *testing.T) {
+	env := map[string]string{}
+
+	fourZeroFour := http.HandlerFunc(func(resWriter http.ResponseWriter, req *http.Request) {
+		resWriter.Write([]byte("Not found"))
+	})
+
+	configurator := yarpcconfig.New(
+		yarpcconfig.InterpolationResolver(mapResolver(env)),
+		yarpcconfig.HTTPFallbackHandler(fourZeroFour),
+	)
+
+	cfgData := map[string]interface{}{
+		"inbounds": map[string]interface{}{
```

Comment:
optional: can use an `type attrs = map[string]interface{}` alias like the
other tests to make this more readable.

```
attrs{
  "inbounds": attrs{
	"http": attrs{"address": "127.0.0.1:8080"},
  },
}
```